modules/base_vpc/outputs.tf

```diff
@@ -23,6 +23,11 @@ output "vpc_endpoint_dynamodb_prefix_list_id" {
   value       = module.vpc_endpoints.endpoints.*.dynamodb[0].prefix_list_id
 }
 
+output "vpc_endpoint_execute_api_prefix_list_id" {
+  description = "The id for the execute API prefix list"
```

Comment:
This is for the audit log api gateway right? Could we add that to the description, so this is more obvious please. If possibly we could even be more explicit with the output name.

Reply:
agree - let's put the term "audit log api gateway" in there somewhere

Reply:
Sure, I'll update now.